charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.2.4
+version: 12.2.5
```

Comment:
Thanks for the PR, please re-base the chart-version and sign the DCO

Reply:
DCO is OK now. Chart version does need another bump though, and conflicts resolved.